Package.swift

```diff
@@ -1,8 +1,34 @@
+// swift-tools-version:4.0
```

Comment:
Removing this line actually causes `swift build` to fail because it's trying to default to the 3.2 tooling (which doesn't understand this package format). I'm still not sure what we _want_ here long term, but I'm thinking we're probably going to end up dropping 3.1 support and pushing forward with the 4.0 style in the future.